src/index.js

```diff
@@ -18,7 +18,14 @@ directory.render(
   <Directory></Directory>
 );
 
-const title = window.location.pathname.split("/").pop();
+let title = window.location.pathname.split("/").pop();
+
+/* For local testing */
+if (title.includes("%20(Prod%20build)%20%7C%7C%20.%20(Local%20development)")) {
```

Comment:
Is this to change the title of the webpage during local testing?

Reply:
Yes, in local testing the webpage has that string added at the front due to how React is configured, I decided to simply filter out the string rather than change the entire configuration while working on it locally.

Reply:
Fair enough, I didn't realize that it could run with that string, I will approve!